msteams-platform/toolkit/debug-overview.md

```diff
@@ -128,60 +124,120 @@ In `.fx/configs/tasks.json` under `"Validate & install prerequisites"` > `"args"
 <details>
 <summary><b>Use your development certificate</b></summary>
 
-1. In `.fx/configs/tasks.json`, uncheck `"devCert"` under `"Validate & install prerequisites"` > `"args"` > `"prerequisites"`.
-1. Set "SSL_CRT_FILE" and "SSL_KEY_FILE" in `.env.teamsfx.local` to your certificate file path and key file path.
+1. In `teamsapp.local.yml`, remove `devCert` from `devTool/install` action (or remove the whole `devTool/install` action if it only contains `devCert`).
+1. In `teamsapp.local.yml`, set `"SSL_CRT_FILE"` and `"SSL_KEY_FILE"` in `file/createOrUpdateEnvironmentFile` action to your certificate file path and key file path.
+
+```yml
+# teamsapp.local.yml
+...
+  # Remove devCert or this whole action
+  - uses: devTool/install
+    with:
+      # devCert:
+  ...
+  - uses: file/createOrUpdateEnvironmentFile
+    with:
+      target: ./.localSettings
+      envs:
+        ...
+        # set your own cert values
+        SSL_CRT_FILE: ...
+        SSL_KEY_FILE: ...
+...
+```
 
 </details>
 
 <details>
-<summary><b>Customize npm install args</b></summary>
+<summary><b>Customize npm install command</b></summary>
 
-In `.fx/configs/tasks.json`, set npmInstallArgs under `"Install npm packages"`.
+In `teamsapp.local.yml`, edit `args` of `cli/runNpmCommand` action.
+
+:::image type="content" source="../assets/images/teams-toolkit-v2/debug/customize-npm-install.png" alt-text="Install npm package":::
   
-   :::image type="content" source="../assets/images/teams-toolkit-v2/debug/customize-npm-install.png" alt-text="Install npm package":::
+```yml
+# teamsapp.local.yml
+...
+  - uses: cli/runNpmCommand
+    with:
+      # edit the npm command args
+      args: install --no-audit
+...
+```
 
 </details>
 
 <details>
 <summary><b>Modify ports</b></summary>
 
 * Bot
-  1. Search for `"3978"` across your project and look for appearances in `tasks.json`, `ngrok.yml` and `index.js`.
+  1. Search for `"3978"` across your project and look for appearances in `tasks.json` and `index.js`.
   1. Replace it with your port.
+
```

Comment:
Replace the screenshot below with:
![image](https://github.com/MicrosoftDocs/msteams-docs/assets/37978464/ac0e51cd-b4ae-4ee2-9972-6d16a87f5977)

Reply:
Updated